test/language/statements/class/subclass/class-definition-null-proto-super.js

```diff
@@ -45,7 +39,7 @@ var reachable = 0;
 class C extends null {
```

Comment:
can this extend Object instead?

Reply:
this test exists to test extending null, the other one is for the generic "not a constructor" case.

Reply:
ah, i see - it was collapsed in the UI. thanks :-)